formatter.R

```diff
@@ -84,6 +84,14 @@ date_type_text <-
     'reported as occurring'
   }
 
+# Adding in NRLS caveat
+nrls_caveat <-
+  if(search_nrls == T & (nrls_decomissioned <= end_date)){
+    '(The NRLS was decommissioned on 30 June 2024)'

```

Comment:
suggest adding a space here 
```suggestion
    ' (The NRLS was decommissioned on 30 June 2024)'

```